### ISSUE TITLE
No encuentro como asociarme o desasociarme de una empresa 

### ISSUE THREAD

**Issue by galloramiro** [PyAr/pyarweb#566]
### Comportamiento actual

No encuentro página con mi perfil o similar que me permita asociarme o desasociarme a una empresa. 

### Comportamiento esperado

Tener una pagina donde pueda desasociarme de la empresa con la que estoy asociado

### Pasos para reproducir el problema

-

### pyAr rev. hash/commit

efd6ffe

### Sistema operativo

Ubuntu

**Comment by tzulberti** [comment]
Lo que se habia discutido es que en esos casos se le manda un mail al admin y el asocia un persona para que sea el admin de la empresa.

**Comment by galloramiro** [comment]
Claro, pero yo cree la empresa, entiendo que sería el admin de la empresa no? y no veo esto. 
Otra pregunta que resalta, yo figuro como admin de la empresa? 
Existe ese roll y esos atributos para los usuarios?

**Comment by facundobatista** [comment]
Para asociarse, hay tres situaciones, creo:
- creás la empresa, deberías quedar asociado a la misma
- buscás la empresa, te aparece alguien que ya está asociado a la misma, te contactás con esa persona
- buscás la empresa pero no hay nadie asociado, mandás mail a `admin@python.org.ar`

Para desasociarte debería un botón de "desasociarme" en la vista de la empresa.

Si hay algo de esto ^ que no anda correctamente, hagámoslo explícito así sabemos en qué trabajar.

Por otro lado, no está el concepto de "admin de la empresa", todos los asociados a la misma son pares.
